### PR TITLE
 Added A Photo Nick to the GCI FOSSASIA Site Student Section 

### LINKED ISSUE #15: Add A Photo Nick to the GCI FOSSASIA Site Student Section
I am working on it .

### PR DESCRIPTION
**Pull request by sahildav**
Fixes #15 
screenshot from 2018-10-24 21-32-50

Link : https://sahildav.github.io/gci18.fossasia.org/

### CODE REVIEW

**Comment by sahildav** [comment]
@abishekvashok  @sk9331657  @ms10398  Please review ??

File: index.html [hunk 1]
```diff
@@ -876,61 +876,52 @@ <h6 class="uppercase mb0">Gill Sans</h6>
             </section>
             <section class="bg-secondary pb0">
                 <div class="container">
-                    <div class="row">
-                        <div class="col-sm-12 text-center">
-                            <h2 class="mb64 mb-xs-24">Flexible, Adaptable & Timeless.</h2>
+                    <div class="row mb64 mb-xs-24">
+                        <div class="col-sm-12 col-md-10 col-md-offset-1 text-center">
+                            <h3>Students Section
```
Comment (abishekvashok) [inline index.html]: Rename to`Students` and close the h3 tag.

Comment (sahildav) [reply index.html]: Done

File: index.html [hunk 2]
```diff
@@ -876,61 +876,52 @@ <h6 class="uppercase mb0">Gill Sans</h6>
...
-                                            <p>
-                                                Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae.
-                                            </p>
+                    </div>
+                    <!--end of row-->
+                    <div class="row masonry masonryFlyIn">
+                        <div class="col-md-4 col-sm-12 masonry-item mb30">
+                           
+                            <div class="col-sm-12 text-center spread-children-large">
+                                    {% for mentor in site.data.students limit:30 %}
```
Comment (sk9331657) [inline index.html]: Change mentor variable to student.

**Comment by sahildav** [comment]
@fragm3 https://sahildav.github.io/gci18.fossasia.org/

**Comment by sahildav** [comment]
@fragm3  Please approve.

**Comment by sk9331657** [comment]
@abishekvashok This can be closed or any other changes are required.

**Comment by sahildav** [comment]
@YashKumarVerma  Please review .

**Comment by sk9331657** [comment]
@fragm3 Reviews??

**Comment by sahildav** [comment]
@ms10398  Please review .

**Comment by sahildav** [comment]
@mariobehling  Please review .

File: index.html [hunk 3]
```diff
@@ -876,61 +876,52 @@ <h6 class="uppercase mb0">Gill Sans</h6>
...
+                    <div class="row masonry masonryFlyIn">
+                        <div class="col-md-4 col-sm-12 masonry-item mb30">
+                           
+                            <div class="col-sm-12 text-center spread-children-large">
+                                    {% for student in site.data.students limit:30 %}
+                                        <div class="card single-mentor">
+                                            {% if student.image %}
+                                                    <img src="img/students/{{student.image}}" alt="{{ student.name }}">
+                                            {% else %}
+                                                <img src="img/students/member.png" alt="{{ student.name }}">
```
[the remark this answers is not in the document]
Comment (sahildav) [reply index.html]: Done Sir.

**Comment by sahildav** [comment]
@akshatnitd  @abishekvashok  Please review and merge this , so that I can proceed with next task.

**Comment by sahildav** [comment]
@abishekvashok @akshatnitd  @fragm3  @YashKumarVerma  Please approve

**Comment by sahildav** [comment]
@akshatnitd  Please merge this.

**Comment by sahildav** [comment]
@mariobehling  Please review this .

**Comment by sahildav** [comment]
@akshatnitd  Thanks , I will make sure for next PR.

**Pull request merged**